README.md

```diff
@@ -185,6 +185,7 @@ e.g `${dirty:-SNAPSHOT}` resolves to `-SNAPSHOT` instead of `-DIRTY`
     - `${version.patch.next}` the `${version.patch}` increased by 1 e.g. '4'
   - `${version.label}` the version label of `${version}` e.g. 'SNAPSHOT'
     - `${version.label.prefixed}` like `${version.label}` with label separator e.g. '-SNAPSHOT'
+    - `${version.label.number}` used to extract the label as a number (build will fail if non-numeric label encountered for this)
```

Comment:
I think we don't need the as number placeholder for the project version label. If you need it please add documentation for `version.label.number.next` and `version.label.number.nextByDistance`

Reply:
following description should be enough
- ${version.label.number} the ${version.label} as number

Reply:
Happy to remove this, will let you know when I've pushed its removal.